kvmagent/kvmagent/test/localstorage_testsuite/test_revert_volume_snapshot_group_with_memory_snapshot.py

```
@@ -0,0 +1,299 @@
+# encoding: utf-8
+import copy
+from unittest import TestCase
+
+import pytest
+
+from kvmagent.test.utils import vm_utils, network_utils, volume_utils, pytest_utils, snapshot_utils
+from kvmagent.test.utils.stub import *
```

Ruff (0.12.2)

`from kvmagent.test.utils.stub import *` used; unable to detect undefined names

(F403)